apple/OmnivoreKit/Sources/Models/PageScrapePayload.swift

```diff
@@ -19,11 +19,18 @@ public struct PageScrapePayload {
   public let url: String
   public let contentType: ContentType
 
-  init(url: String, title: String?, html: String?) {
+  init(url: String, title: String?, html: String?, contentType: String?) {
     self.url = url
     self.title = title
     self.html = html
-    self.contentType = url.hasSuffix(".pdf") ? .pdf : .html
+
+    // If the content type was specified and we know its PDF, use that
+    // otherwise fallback to using file extensions.
+    if let contentType = contentType, contentType == "application/pdf" {
```

Comment:
should we use `contentType.contains("pdf")` just in case we get a site that has the mime type wrong?

Reply:
From what I can tell this is set by Safari not straight from the site headers, but yeah that seems a bit safer.